### PR TITLE
Added Glasgow table

### PR DESCRIPTION
**Pull request by Sandra3657**
Related to #1576
The glasgow table and description has been added.

image

### CODE REVIEW

**Comment by nihal467** [comment]
eroor

test failed, remove the '_' in between the words

**Comment by Sandra3657** [comment]
This will be fixed by PR #1644.

File: src/Components/Facility/Consultations/GlasgowTables.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    { value: 3, text: "Inappropriate words/Cry to Pain" },
+    { value: 2, text: "Incomprehensible words/Moans to pain" },
+    { value: 1, text: "None" },
+  ];
+
+  const MOTOR_RESPONSE_SCALE = [
+    { value: 6, text: "Obeying commands" },
+    { value: 5, text: "Moves to localised pain" },
+    { value: 4, text: "Flexion withdrawal from pain" },
+    { value: 3, text: "Abnormal Flexion(decorticate)" },
```
Comment (ghost) [inline src/Components/Facility/Consultations/GlasgowTables.tsx]: can you move these constants in `constants.ts` where already many constants are defined.

**Review by abhinandanarya06 — APPROVED** [review]
Resolve the below suggestions and merge conflicts. 
remaining LGTM. After changes, i will merge your branch in my branch and do necessary changes 👍🏻

**Comment by nihal467** [comment]
kpffkwep fewk

@Sandra3657 the latest result is coming in the rightmost of the list, so when a lot of data comes in, to see the recent result, the viewer has to scroll rightwards, so I think the lastest result would be nice to come on the leftmost, so the viewer can see the latest result as first in the list

@bodhish what do you think?

**Review by peeyush14goyal — APPROVED** [review]

**Comment by Sandra3657** [comment]
The scrollbar is default moved to the right first, So that the latest data can be seen first. The user needs to move the scrollbar left to see the previous results.

**Pull request merged**